pkg/install/kubeconfig.go

```diff
@@ -0,0 +1,93 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"reflect"
+	"time"
+
+	"github.com/ghodss/yaml"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/kubeconfig"
+	"github.com/openshift/installer/pkg/asset/tls"
+	clientcmd "k8s.io/client-go/tools/clientcmd/api/v1"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+// generateAndStoreKubeconfigs generates additional admin credentials and kubeconfig based admin kubeconfig
+// found in graph and stores both to the database
+func (i *Installer) generateAndStoreKubeconfigs(ctx context.Context, g graph, aroServiceName string) error {
+
+	ca := g[reflect.TypeOf(&tls.AdminKubeConfigSignerCertKey{})].(*tls.AdminKubeConfigSignerCertKey)
+	clientCertKey := g[reflect.TypeOf(&tls.AdminKubeConfigClientCertKey{})].(*tls.AdminKubeConfigClientCertKey)
```

Comment:
I think you can just say `var clientCertKey tls.SignedCertKey` instead of this line - move to around line 38